README.md

```diff
@@ -65,27 +69,64 @@ This paragraph is only a quickstart guide for the compilation of
 Centreon Engine. For a more in-depth guide with build options you should
 refer to the [online documentation](https://documentation.centreon.com/docs/centreon-engine/en/latest/installation/index.html#using-sources).
 
-Centreon Engine needs Centreon Clib to be build. You should
-[install it first](https://github.com/centreon/centreon-clib).
+**For Centos 7 :**
+First of all, check if you have these packages installed :
+
+    git, make, cmake, gcc-c++, python3, epel-release, cmake3
+
+If they are not installed, please install them.
+
+Conan is needed as well but make sure to install gcc before conan.
+When you are ready do :  
+    $> pip3 install conan  
+
+Then you will be able to resume the following steps :
+    $> git clone https://github.com/centreon/centreon-engine
+    $> cd centreon-engine && ./cmake.sh
+    $> cd build
+    $> make && make install
+
+
+**For other didtributions :**
+If you are on another distribution, then follow the steps below.
+
+Check if you have these packages installed (Note that packages names come from Centos 7 distribution, so if some packages names don't match on your distribution try to find their equivalent names) :
+
+    git, make, cmake, gcc-c++, python3, cmake3
 
-Once the sources of Centreon Engine extracted go to the *./build/*
-directory and launch the CMake command. This will look for required
-dependencies and print a summary of the compilation parameters if
-everything went fine.
+For the projet compilation you need to have conan installed. Try to use the package manager given by your OS to install conan. ('apt' for Debian, 'rpm' for Red Hat, 'pacman' for Arch Linux, ...) It is prefered to install gcc before conan.
 
+Exemple :
+    $> _apt_  install conan
+
+If it does not work, conan can be installed with pip3 :
+    $> pip3 install conan
+
+You can now compile centreon-engine :
+    $> git clone https://github.com/centreon/centreon-engine
     $> cd centreon-engine/build
```

Comment:
the build directory is not here by default you can replace 
$> cd centreon-engine/build
by
$> mkdir -p centreon-engine/build
$> cd centreon-engine/build